third_party/libuv_test.patch

```diff
@@ -0,0 +1,56 @@
+diff --git a/test/test-timer.c b/test/test-timer.c
+index c667da00..58693ac2 100644
+--- a/test/test-timer.c
++++ b/test/test-timer.c
+@@ -25,6 +25,8 @@
+ 
+ static int once_cb_called = 0;
+ static int once_close_cb_called = 0;
++static int twice_cb_called = 0;
++static int twice_close_cb_called = 0;
+ static int repeat_cb_called = 0;
+ static int repeat_close_cb_called = 0;
+ static int order_cb_called = 0;
+@@ -58,6 +60,27 @@ static void once_cb(uv_timer_t* handle) {
+   uv_update_time(uv_default_loop());
+ }
+ 
++static void twice_close_cb(uv_handle_t* handle) {
++  printf("TWICE_CLOSE_CB\n");
++
++  ASSERT(handle != NULL);
++  ASSERT(0 == uv_is_active(handle));
++
++  twice_close_cb_called++;
++}
++
++static void twice_cb(uv_timer_t* handle) {
++  printf("TWICE_CB %d\n", twice_cb_called);
++
++  ASSERT(handle != NULL);
++  ASSERT(0 == uv_is_active((uv_handle_t*) handle));
++
++  twice_cb_called++;
++
++  uv_close((uv_handle_t*)handle, twice_close_cb);
++}
++
++
+ 
+ static void repeat_close_cb(uv_handle_t* handle) {
+   printf("REPEAT_CLOSE_CB\n");
+@@ -144,12 +167,12 @@ TEST_IMPL(timer_start_twice) {
+   ASSERT(r == 0);
+   r = uv_timer_start(&once, never_cb, 86400 * 1000, 0);
+   ASSERT(r == 0);
+-  r = uv_timer_start(&once, once_cb, 10, 0);
++  r = uv_timer_start(&once, twice_cb, 10, 0);
+   ASSERT(r == 0);
+   r = uv_run(uv_default_loop(), UV_RUN_DEFAULT);
+   ASSERT(r == 0);
+ 
+-  ASSERT(once_cb_called == 1);
++  ASSERT(twice_cb_called == 1);
```

Comment:
This change could be a bit misleading. The purpose of this test is that `once_cb` is called exactly once, even when started twice. Calling it `twice_cb` maybe implies that it's somehow different from `once_cb`, which it should not be.

Reply:
It’s for test case `timer_start_twice`, which is the named similar to other xx_cb variables. Can we keep it for now and let the libuv team decide when I send them the PR?